### PR TITLE
Pol 554 AWS RI and Savings Plan Currency Adjustments

### PR DESCRIPTION
**Pull request by lawrethao**
### Description

Adds currency parameter to aws instance and savings recommendations to multiply against monthly estimated savings

### Issues Resolved

fixes issue of not being able to convert savings into usable value

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by cdwilhelm — CHANGES_REQUESTED** [review]
thanks for getting this done. some questions that may lead to change.

File: cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt [hunk 1]
```diff
@@ -85,6 +85,15 @@ parameter "param_email" do
   description "Email addresses of the recipients you wish to notify"
 end
 
+parameter "param_currency_adjustment" do
+  label "Currency Adjustment"
+  description "adjusts recommendation savings based on USD conversion rate where USD is 100% - Maximum 5.0"
+  default 1.00
+  min_value 0
```
[the remark this answers is not in the document]
Comment (lawrethao) [reply cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt]: I've removed the minimum value. do you know the use case? I couldn't think of when you'd have a currency that is a negative conversion number if value is based on a percentage against the USD.

**Review by cdwilhelm — APPROVED** [review]
LGTM

**Comment by douglaswth** [comment]
@cdwilhelm / @lawrethao This PR only updated the version and not in the actual `info` metadata of the Policy Templates, so the new versions have not been published to the catalog.

**Comment by cdwilhelm** [comment]
@douglaswth  thanks for catching this.  we'll get it fixed up

**Pull request merged**